esmvalcore/config-developer.yml

```diff
@@ -191,10 +191,10 @@ OBS6:
 native6:
   cmor_strict: false
   input_dir:
-    default: 'Tier{tier}/{dataset}'
+    default: '{dataset}/{latestversion}/{frequency}/{short_name}'
```

Comment:
Why do we need `{latestversion}` here?

Reply:
Also, wouldn't be more straightforward to use `era5_name` instead of `short_name`?

Reply:
> Why do we need {latestversion} here?

Because we want to support multiple versions of datasets

> Also, wouldn't be more straightforward to use `era5_name` instead of `short_name`?

Using the short name has the advantage that you can also use this DRS for other observational datasets than ERA5, making it easier to add support for those.

Reply:
OK. I think I understand why it didn't work before.
Let me try again.

Reply:
I provided an example of how to use it in the issue (#641), maybe that helps?

Reply:
Can we add `Tier{tier}/`, I think it would be good to keep this information in the directory tree.

Reply:
Do we really need those Tier1, 2, 3 directories? Is it possible for one dataset to be in two different Tiers?

Reply:
> Do we really need those Tier1, 2, 3 directories? 
Yes, we should keep this classification, it's also described in the paper.

> Is it possible for one dataset to be in two different Tiers?
No, I don't think so.

I can also add another entry for dkrz with `Tier{tier}` and leave `default` unchanged.

Do we still need the `era5cli` entry? I think that would never work given the ambiguity in variable name.

Reply:
> I can also add another entry for dkrz with Tier{tier} and leave default unchanged.

Then we still need it in the recipes, so it would not help. Let's keep it then.

> Do we still need the era5cli entry? I think that would never work given the ambiguity in variable name.

I think it can be deleted, I guess I would need to add some documentation somewhere then on what directory structure to use.